sdk/src/lib.rs

```diff
@@ -123,33 +121,24 @@ impl ProverClient {
         let proof_id = client.create_proof(elf, &stdin).await?;
         println!("proof_id: {:?}", proof_id);
 
-        let mut pb = StageProgressBar::new();
```

Comment:
Why did we remove the progress bar?

Reply:
the network no longer has information about how far a proof has progressed